appengine/bigquery/README.md

```diff
@@ -0,0 +1,168 @@
+# Accessing BigQuery from App Engine
+
+A Google Cloud Platform customer asked me today how to list all the BigQuery
+projects that you own from a Google App Engine app.
```

Comment:
This reads like a blog post. You may want to add your name above, or re-word so it's more like a normal README.